README.md

```diff
@@ -2,130 +2,134 @@
 
 # Kata Containers
 
-* [Raising issues](#raising-issues)
-* [Kata Containers repositories](#kata-containers-repositories)
-    * [Code Repositories](#code-repositories)
-        * [Kata Containers-developed components](#kata-containers-developed-components)
-            * [Agent](#agent)
-            * [KSM throttler](#ksm-throttler)
-            * [Runtime](#runtime)
-            * [Trace forwarder](#trace-forwarder)
-        * [Additional](#additional)
-            * [Kernel](#kernel)
-    * [CI](#ci)
-    * [Community](#community)
+* [Kata Containers](#kata-containers)
+    * [Introduction](#introduction)
+    * [Getting started](#getting-started)
     * [Documentation](#documentation)
-    * [Packaging](#packaging)
-    * [Test code](#test-code)
-    * [Utilities](#utilities)
-        * [OS builder](#os-builder)
-    * [Web content](#web-content)
+    * [Community](#community)
+    * [Getting help](#getting-help)
+        * [Raising issues](#raising-issues)
+            * [Current version](#current-version)
+            * [Kata Containers 1.x legacy versions](#kata-containers-1x-legacy-versions)
+    * [Developers](#developers)
+        * [Components](#components)
+            * [Legacy components](#legacy-components)
+        * [Common repositories](#common-repositories)
+    * [Credits](#credits)
 
 ---
 
 Welcome to Kata Containers!
 
-The purpose of this repository is to act as a "top level" site for the project. Specifically it is used:
-
-- To provide a list of the various *other* [Kata Containers repositories](#kata-containers-repositories),
-  along with a brief explanation of their purpose.
-
-- To provide a general area for [Raising Issues](#raising-issues).
+This repository is the home of the current Kata Containers code.
 
-## Raising issues
+If you want to learn about Kata Containers, visit the main
+[Kata Containers website](https://katacontainers.io).
 
-This repository is used for [raising
-issues](https://github.com/kata-containers/kata-containers/issues/new):
-
-- That might affect multiple code repositories.
-
-- Where the raiser is unsure which repositories are affected.
-
-> **Note:**
-> 
-> - If an issue affects only a single component, it should be raised in that
->   components repository.
+For further details on the Kata Containers 1.x legacy versions, see the
```

Comment:
I am bit apprehensive of using legacy here. I feel we should not call 1.x as legacy just yet, atleast till be announce it as deprecated. wdyt?

Reply:
I've removed the word 'legacy' and added references to 1.xand/or  "first generation".

Reply:
It's worth pointing out that we already use the term "legacy" for 1.x though:

- https://github.com/kata-containers/kata-containers/tree/2.0-dev/docs/install#legacy-installation.